index.html

```diff
@@ -499,511 +355,1145 @@ <h3>Multikey</h3>
+          </p>
+          <ol>
+            <li>
+The issuers private/public key pair, i.e., the key pair corresponding to the
+verification method that will be part of the proof.
+            </li>
+            <li>
+An HMAC key. This used to randomize the order of the blank node ids to avoid
+potential information leakage from the blank node id ordering. This is used only
+once and is shared between issuer and holder. The HMAC in this case is
+functioning as a pseudorandom function (PRF).
+            </li>
+          </ol>
+          <p>
+The key material used for generating the add base proof test vectors is shown
+below. Hexadecimal representation is use for the BBS key pairs and the HMAC key.
+          </p>
+          <pre class="example nohighlight" title="Private and Public keys for Signature"
+          data-include="TestVectors/BBSKeyMaterial.json"
+          data-include-format="text">
+          </pre>
+          <p>
+In our scenario a sailor is registering with a race organizer for a series of
+windsurfing races to be held over a number of days on Maui. The organizer will
+inspect the sailors equipment to certify that what has been declared is
+  accurate. The sailors unsigned equipment inventory is shown below.
+          </p>
+          <pre class="example nohighlight" title="Credential without Proof" data-include="TestVectors/windDoc.json"
+          data-include-format="text"></pre>
+          <p>
+In addition to let other sailors know what kinds of equipment their competitors
+maybe sailing on it is mandatory that each sailor disclose the year of their
+most recent windsurfing board and full details on two of their sails. Note that
+all sailors are identified by a sail number that is printed on all their
+equipment. This mandatory information is specified via an array of JSON pointers
+as shown below.
+          </p>
+          <pre class="example nohighlight" title="Mandatory Pointers" data-include="TestVectors/windMandatory.json"
+          data-include-format="text"></pre>
+          <p>
+The result of applying the above JSON pointers to the sailors equipment document
+is shown below.
+          </p>
+          <pre class="example nohighlight" title="JSON Pointers and Values" data-include="TestVectors/addPointerValues.json"
+          data-include-format="text"></pre>
+          <p>
+Transformation of the unsigned document begins with canonicalizing the document
+as shown below.
+          </p>
+          <pre class="example nohighlight" title="Canonical Document" data-include="TestVectors/addBaseDocCanon.json"
+          data-include-format="text"></pre>
+          <p>
+To prevent possible information leakage from the ordering of the blank node ids
+these are processed through a PRF, i.e., the HMAC to give the canonized HMAC
+document shown below. This represents an ordered list of statements that will be
+subject to mandatory and selective disclosure, i.e., it is from this list that
+statements are grouped.
```

Comment:
```suggestion
To prevent possible information leakage from the ordering of the blank node IDs
these are processed through a PRF (i.e., the HMAC) to give the canonicalized HMAC
document shown below. This represents an ordered list of statements that will be
subject to mandatory and selective disclosure, i.e., it is from this list that
statements are grouped.
```